Test/TestConfigurationUtils.ps1

```diff
@@ -165,9 +165,14 @@ function Stop-DockerDriver {
 
     Invoke-Command -Session $Session -ScriptBlock {
         Stop-Service docker | Out-Null
-        Get-NetNat | Remove-NetNat -Confirm:$false
-        Get-ContainerNetwork | Remove-ContainerNetwork -ErrorAction SilentlyContinue -Force
-        Get-ContainerNetwork | Remove-ContainerNetwork -Force
+
+        Get-NetNat | ForEach-Object {
```

Comment:
We should put a comment describing why we remote NAT networks and container networks this way.